pkgs/tools/filesystems/romdirfs/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, fuse }:
+
+stdenv.mkDerivation rec {
+  pname = "romdirfs";
+  version = "1.2";
+
+  src = fetchFromGitHub {
+    owner = "mlafeldt";
+    repo = "romdirfs";
+    rev = "v${version}";
+    sha256 = "1jbsmpklrycz5q86qmzvbz4iz2g5fvd7p9nca160aw2izwpws0g7";
+  };
+
+  buildInputs = [ cmake fuse pkgconfig ];
```

Comment:
`cmake` and `pkgconfig` should be in `nativeBuildInputs`.

Reply:
So right, ho i missed it.